enhancements/network/ovs-dynamic-cpu-pinning.md

```diff
@@ -0,0 +1,129 @@
+---
+title: ovs-dynamic-cpu-pinning
+authors:
+  - "@zeeke"
+reviewers:
+  - "@trozet"
+  - "@MarSik"
+  - "@rycky-rav"
+  - "@cgonclaves"
```

Comment:
```suggestion
  - "@cgoncalves"
```

Reply:
`ricky-rav` :)   or `rravaiol` on the company register

Reply:
sorry! fixing